app/classifier/index.cjsx

```diff
@@ -17,12 +17,17 @@ workflowAllowsSeparateFrames = require '../lib/workflow-allows-separate-frames'
 WorldWideTelescope = require './world_wide_telescope'
 MiniCourseButton = require './mini-course-button'
 GridTool = require './drawing-tools/grid'
+Intervention = require '../lib/intervention'
+experimentsClient = new (require '../lib/experiments-client')
+
+PULSAR_HUNTERS_SLUG = 'zooniverse/pulsar-hunters'
```

Comment:
yeah i had some confusion over that line and some other parts while rebasing, i know it's not quite right yet. will clean up before moving out of WIP state.